src/saltext/vmware/modules/esxi.py

```diff
@@ -698,6 +698,210 @@ def get_firewall_config(
         raise salt.exceptions.SaltException(str(exc))
 
 
+def get_firewall_config(
```

Comment:
So above on line 628 we have get_firewall_config -> get_firewall_configs, and here we have get_firewall_config.

If this is a typo, needs fixing, but don't think so due to ruleset_name, then this function is poorly named, since a simple typo and dropped 's' and boom !!!,. not what you expect.